scripts/qa_crowbarsetup.sh

```diff
@@ -622,6 +622,8 @@ function onadmin_prepare_cloud_repos()
             develcloud5)
                 addsp3testupdates
                 add_sles12ga_testupdates
+                addcloud5maintupdates
```

Comment:
I'm not sure I understand the question about GM5: there's no change for this compared to what we do for Cloud 3 / Cloud 4.

As to the develcloud5 case, what about disabling the repo in that case?